Typings for modifyEager do not reflect documentation

It looks like the typings for [`modifyEager`](https://vincit.github.io/objection.js/#modifyeager) are slightly out of date when compared with the documentation. [According to the docs](https://vincit.github.io/objection.js/#modifiers), the following is allowed:

```ts
Person
  .query()
  .eager('[movies.actors, pets]')
  .modifyEager('movies', ['goodMovies', 'orderByName'])
  .modifyEager('pets', 'dogs')
```

Notice the second argument to `modifyEager` can also be `string | string[]`

TypeScript raises an error when it is used like this:

```
error TS2345: Argument of type 'string[]' is not assignable to parameter of type '(builder: QueryBuilder<Model, Model[], Model[]>) => void'.
  Type 'string[]' provides no match for the signature '(builder: QueryBuilder<Model, Model[], Model[]>): void'.

11       .modifyEager('movies', ['goodMovies', 'orderByName']);
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

The code runs just fine when using `string | string[]`

---

If this issue is valid, I am happy to open a PR to correct the typings.

--- a/tests/ts/examples.ts
+++ b/tests/ts/examples.ts
@@ -513,7 +513,9 @@ const children: Promise<Person[]> = Person.query()
 const childrenAndPets: Promise<Person[]> = Person.query()
   .eager('children')
   .where('age', '>=', 42)
-  .modifyEager('[pets, children.pets]', qb => qb.orderBy('name'));
+  .modifyEager('[pets, children.pets]', qb => qb.orderBy('name'))
+  .modifyEager('[pets, children.pets]', 'orderByName')
+  .modifyEager('[pets, children.pets]', ['orderByName', 'orderBySomethingElse']);
 
 const rowsPage: Promise<{
   total: number;

--- a/typings/objection/index.d.ts
+++ b/typings/objection/index.d.ts
@@ -368,6 +368,10 @@ declare namespace Objection {
       relationExpression: RelationExpression,
       modifier: (builder: QueryBuilder<QM2, QM2[]>) => void
     ): QueryBuilder<QM1, RM1, RV1>;
+    <QM2 extends Model>(
+      relationExpression: RelationExpression,
+      modifier: string | string[]
+    ): QueryBuilder<QM1, RM1, RV1>;
   }
 
   interface BluebirdMapper<T, Result> {

